### ISSUE TITLE
Does *All Accessibility Metadata* means all onix list 196 codes ?

### ISSUE THREAD

**Issue by editadapt** [w3c/publ-a11y#81]
In the [*User Experience Guide for Displaying Accessibility Metadata 1.0* point 4.9](https://www.w3.org/2021/09/UX-Guide-metadata-1.0/principles/#all-accessibility-metadata) and  [*Display Techniques for ONIX Accessibility Metadata 1.0 point 2.9*](https://www.w3.org/publishing/a11y/UX-Guide-metadata/techniques/onix-metadata/index.html#all-accessibility-metadata) we find : All Accessibility Metadata Value: Link to complete list of all metadata fields

There is a link to a full [EPUB Accessibility exemple](https://www.w3.org/publishing/a11y/UX-Guide-metadata/techniques/epub-metadata/index.html#all-accessibility-metadata)

For ONIX we have the following indication : [For a complete list of ONIX accessibility metadata refer to the crosswalk.](http://www.a11ymetadata.org/the-specification/metadata-crosswalk/) wich is now outdated. 

1. The ONIX link should now link to [the updated Crosswalk between metadata formats.](https://w3c.github.io/a11y-discov-vocab/crosswalk/)
2. Do we have to show all ONIX list 196 codes (if presents) ? Do we have to show only the ones with a schema dot org reference ? Am I missing something ?

**Comment by avneeshsingh** [comment]
"All accessibility metadata" looks as misleading term here. We cannot expect that publishers would be providing all accessibility metadata of ONIX codepage 196 or all accessibility metadata fields of schema.org.
The intent here is to show all the accessibility metadata of the publication.
